packages/types/api/auth.ts

```
@@ -6,6 +6,21 @@ export type LoginRes = Common<{
 
 export type UserInfoRes = Common<{
-  account_type: 'USER' | 'ADMIN'
+  account_type: 'USER' | 'ADMIN' | 'GUEST'
   name: string
 }>
+
+export type SendAuthCodeRes = Common<{
+  try_cnt: number
+}>
+
+export type VerifyAuthCodeRes = Common<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)